repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/tableauLookupSection.ftl

```diff
@@ -47,6 +47,7 @@
 
        <div id="${divId}" class="viz-wrapper">
            <div id="${divId}-viz" class="viz-wrapper-item"></div>
+           <div id="downLoadData"></div>
```

Comment:
Please update "downLoadData" to '${divId}-downLoadData" as there can be more than one instance of Tableau per page.